bootstrapper/forms/formsBootstrapper.ts

```diff
@@ -1,13 +1,25 @@
 import * as angular from 'angular';
 
+import { services } from 'typescript-angular-utilities';
+import __validation = services.validation;
+
 export const moduleName: string = 'FormTestModule';
 
 class FormTestController {
 	count: number;
+	text: string;
+	validator: __validation.IValidationHandler;
 
 	static $inject: string[] = ['$q', '$timeout'];
 	constructor(private $q: angular.IQService, private $timeout: angular.ITimeoutService) {}
 
+	$onInit(): void {
+		this.validator = {
+			validate: () => this.text === 'valid',
```

Comment:
That's a fair question. The purpose of the bootstrapper is different, since it's designed to demonstrate and test functionality. We could certainly change this to some other validator, like string length > 5-10 or something.